airflow/migrations/versions/0151_2_10_0_dataset_alias_unique_constraint.py

```diff
@@ -0,0 +1,49 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Add unique constraint to dataset_alias table.
+
+Revision ID: d57c3e740878
+Revises: 8684e37832e6
+Create Date: 2024-07-29 12:59:07.793836
+
+"""
+
+from __future__ import annotations
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "d57c3e740878"
+down_revision = "8684e37832e6"
+branch_labels = None
+depends_on = None
+airflow_version = "2.10.0"
+
+
+def upgrade():
+    """Add unique constraint to dataset_alias table."""
+    with op.batch_alter_table("dataset_alias", schema=None) as batch_op:
+        batch_op.create_index("idx_name_unique", ["name"], unique=True)
```

Comment:
Since we haven't released the previous migration, can we update this in that migration instead of a new migration file

Reply:
Thanks! Didn't know we should do that. Just updated it